apps/challenges/views.py

```diff
@@ -3229,6 +3229,35 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                         }
                         send_slack_notification(message=message)
 
+                    template_data = get_challenge_template_data(zip_config.challenge)
+                    if (
+                        not challenge.is_docker_based
+                        and challenge.inform_hosts
+                        and not challenge.remote_evaluation
```

Comment:
@afif-fahim we need to add a check to start worker only if this API was called for the first time. You can use the value of the number of workers in challenge model to figure out if we have already started the worker for a challenge or not. If value is not null then we don't need to call the `start_worker` method

Reply:
@Ram81 can you review again please?